app/js/templates.js

```diff
@@ -3,9 +3,20 @@ var lineno = null;
 var colno = null;
 var output = "";
 try {
-output += "<div class=\"Comment-content\">";
-output += runtime.suppressValue(runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "comment")),"content", env.autoesc), env.autoesc);
-output += "</div>\n\n<div class=\"Comment-meta hbox\">\n  ";
+output += "<div class=\"Comment-content\">\n  ";
+if(runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "comment")),"title", env.autoesc)) {
+output += "\n    ";
+output += runtime.suppressValue(env.getFilter("escape").call(context, runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "comment")),"title", env.autoesc)), env.autoesc);
+output += "\n  ";
+;
+}
+else {
+output += "\n    ";
+output += runtime.suppressValue(env.getFilter("safe").call(context, runtime.memberLookup((runtime.contextOrFrameLookup(context, frame, "comment")),"content", env.autoesc)), env.autoesc);
+output += "\n  ";
+;
+}
+output += "\n</div>\n\n<div class=\"Comment-meta hbox\">\n  ";
```

Comment:
I just haven't paid attention at first. And when we started suggesting installing by getting the ZIP from Github, I didn't want to break people's flow.